src/js/components/Tabs/Tabs.js

```diff
@@ -9,109 +9,98 @@ import { Box } from '../Box';
-    delete rest.onActive;
-    const { activeIndex } = this.state;
-
-    let activeContent;
-    let activeTitle;
-    const tabs = Children.map(
-      children,
-      (tab, index) => {
-        if (!tab) return undefined;
-
-        const tabProps = tab.props || {};
-
-        const isTabActive = index === activeIndex;
-
-        if (isTabActive) {
-          activeContent = tabProps.children;
-          if (typeof tabProps.title === 'string') {
-            activeTitle = tabProps.title;
-          } else {
-            activeTitle = index + 1;
-          }
-        }
+  /* eslint-disable no-param-reassign */
+  delete rest.activeIndex;
+  delete rest.onActive;
+  /* eslint-enable no-param-reassign */
 
-        return cloneElement(tab, {
-          active: isTabActive,
-          onActivate: () => this.activateTab(index),
-        });
-      },
-      this,
-    );
+  let activeContent;
+  let activeTitle;
+  const tabs = Children.map(
+    children,
+    (tab, index) => {
+      if (!tab) return undefined;
 
-    const tabContentTitle = `${activeTitle || ''} ${tabContents}`;
+      const tabProps = tab.props || {};
 
-    return (
-      <StyledTabs
+      const isTabActive = index === activeIndex;
+
+      if (isTabActive) {
+        activeContent = tabProps.children;
+        if (typeof tabProps.title === 'string') {
+          activeTitle = tabProps.title;
+        } else {
+          activeTitle = index + 1;
+        }
+      }
+
+      return cloneElement(tab, {
+        active: isTabActive,
+        onActivate: () => activateTab(index),
+      });
+    },
+    this,
+  );
+
+  const tabContentTitle = `${activeTitle || ''} ${tabContents}`;
+
+  return (
+    <StyledTabs
+      as={Box}
+      role="tablist"
+      flex={flex}
+      {...rest}
+      background={theme.tabs.background}
+    >
+      <StyledTabsHeader
         as={Box}
-        role="tablist"
-        flex={flex}
-        {...rest}
-        background={theme.tabs.background}
+        direction="row"
+        justify={justify}
+        flex={false}
+        wrap
+        background={theme.tabs.header.background}
+        gap={theme.tabs.gap}
       >
-        <StyledTabsHeader
-          as={Box}
-          direction="row"
-          justify={justify}
-          flex={false}
-          wrap
-          background={theme.tabs.header.background}
-          gap={theme.tabs.gap}
-        >
-          {tabs}
-        </StyledTabsHeader>
-        <StyledTabPanel
-          flex={flex}
-          aria-label={tabContentTitle}
-          role="tabpanel"
-        >
-          {activeContent}
-        </StyledTabPanel>
-      </StyledTabs>
-    );
-  }
-}
+        {tabs}
+      </StyledTabsHeader>
+      <StyledTabPanel flex={flex} aria-label={tabContentTitle} role="tabpanel">
+        {activeContent}
+      </StyledTabPanel>
+    </StyledTabs>
+  );
+};
+
+Tabs.defaultProps = {
+  justify: 'center',
+  messages: {
+    tabContents: 'Tab Contents',
+  },
+  responsive: true,
```

Comment:
I don't think this prop is being used, can you take a look if we can clean it out and remove all the Tabs.defaultProps block?

Reply:
Yea, it was just being forwarded to the `StyledTabs` and we are not using it there. I removed it.

Reply:
One question. We are importing `{ defaultProps } from '../../default-props';` and were merging it with `Object.setPrototypeOf(Tabs.defaultProps, defaultProps);`

Now that we've removed the component's defaultProps declaration block, how are we going to merge the imported `defaultProps`?

One by one? like `theme = defaultProps.theme`? Or is there any better way?

Reply:
Good catch.
I'm thinking something like 
```
Object.setPrototypeOf({}, defaultProps);
```
Thoughts?

Reply:
By doing that, we'd be setting the `defaultProps` on an empty object, not `Tabs.defaultProps`.
Am I overthinking? I didn't test it by the way.

Reply:
I think you are right.
```
Tabs.defaultProps = {};
Object.setPrototypeOf(Tabs.defaultProps, defaultProps);
```

Reply:
Thanks, made the change.